test/Microsoft.AspNetCore.Mvc.Core.Test/ModelBinding/Binders/EnumTypeModelBinderProviderTest.cs

```diff
@@ -15,7 +16,7 @@ public class EnumTypeModelBinderProviderTest
         public void ReturnsBinder_ForEnumType(Type modelType)
         {
             // Arrange
-            var provider = new EnumTypeModelBinderProvider(allowBindingUndefinedValueToEnumType: true);
+            var provider = new EnumTypeModelBinderProvider(new TestMvcOptions(allowBindingUndefinedValueToEnumType: true).Value);
```

Comment:
`Options.Create(new MvcOptions { AllowBindingUndefinedValueToEnumType = true });`

Reply:
Why not `... new EnumTypeModelBinderProvider(new MvcOptions { AllowBindingUndefinedValueToEnumType = true });`?